src/main/java/org/breedinginsight/brapi/v2/services/BrAPIGermplasmService.java

```diff
@@ -149,8 +131,37 @@ public DownloadFile exportGermplasmList(UUID programId, String listId) throws Ap
 
             processedData.add(row);
         }
+        return processedData;
+    }
+
+    public DownloadFile exportGermplasmList(UUID programId, String listId, FileType fileExtension) throws ApiException, IOException {
+        List<Column> columns = GermplasmFileColumns.getOrderedColumns();
+
+        //Retrieve germplasm list data
+        BrAPIListDetails listData = brAPIListDAO.getListById(listId, programId).getResult();
+
+        //Retrieve germplasm data
+        List<String> germplasmNames = listData.getData();
+        List<BrAPIGermplasm> germplasm = germplasmDAO.getGermplasmByRawName(germplasmNames, programId);
+        //processGermplasmForDisplay, numbers
+        germplasm.sort(Comparator.comparingInt(g -> g.getAdditionalInfo().get("importEntryNumber").getAsInt()));
```

Comment:
Chris did some work in #192 to refactor the string literals in the `additionalInfo` object to be constants.  It may be worth coordinating with him on how to go about merging to pick up that change

Reply:
I am going to implement the constants here after Heather merges this in since her merge will happen before mine.